doc/articles/interop/apple-login.md

```diff
@@ -0,0 +1,114 @@
+---
+uid: Uno.Tutorials.SignInWithApple
+---
+
+# Apple Sign-In Button with MAUI Embedding in Uno
+
+In Uno Platform projects, implementing a "Sign in with Apple" Button is possible through MAUI embedding (more on which in our [Overview](xref:Uno.Extensions.Maui.Overview)). Below is a simple example of how you can add the Apple Sign-In Button to your project. We will inject it into the visual tree from the code-behind and handle the authorization using Apple's Authentication Services.
+
+## Steps to Add "Sign in with Apple" Button
+
+Below is a simple example of how you can add the Apple Sign-In Button to your project. We will inject it into the visual tree from the code-behind and handle the authorization using Apple's Authentication Services.
+
+### General usage
+
+1. Import necessary namespaces:
+
+   ```csharp
+   using AuthenticationServices;
+   using Foundation;
+   using UIKit;
+   ```
+
+2. In the constructor of your Page or control, create the Apple Sign-In Button:
+
+   ```csharp
+   var appleSignInButton = new ASAuthorizationAppleIdButton(ASAuthorizationAppleIdButtonType.Default, ASAuthorizationAppleIdButtonStyle.WhiteOutline);
+   appleSignInButton.TouchUpInside += HandleAuthorizationAppleIDButtonPress;
+   appleSignInButton.CornerRadius = 50;
```

Comment:
We also need to create a the delegate, which will handle when the native SignInWithApple flow completes:

```_appleSignInDelegate = new AuthorizationControllerDelegate(this);```

I'm not sure if this needs to be a member variable, but we kept a reference to it just in case to prevent garbage collection. 

If they're using MVVM, this would probably be where they pass their view model or whatever service they want to call after the sign in completes. We just pass our user control so that the delegate can call a function on it (we're not using MVVM nor MVUX).

Reply:
Thank you for the suggestions! I have made some changes, let me know what you think